src/fabric/src/fabric2_active_tasks.erl

```diff
@@ -0,0 +1,28 @@
+-module(fabric2_active_tasks).
+
+
+-export([
+    get_active_tasks/0
+]).
+
+
+-include_lib("couch/include/couch_db.hrl").
+-include("fabric2.hrl").
+
+
+-define(ACTIVE_TASK_JOB_TYPES, [<<"views">>, <<"replication">>]).
```

Comment:
To avoid hard-coding the types we could just let any job expose the active_task_info object and we'd automatically display it. So in the get_active_tasks we could query all the types first then call get_active_jobs(JTx, Type).

To get all the types we could expose https://github.com/apache/couchdb/blob/prototype/fdb-layer/src/couch_jobs/src/couch_jobs_fdb.erl#L321 in `couch_jobs` as 

```
get_types(Tx) ->
     couch_jobs_fdb:tx(couch_jobs_fdb:get_jtx(Tx), fun(JTx) ->
        couch_jobs_fdb:get_types(JTx)
    end).
```

This fabric app doesn't have to know anything about replication or views and simply reports status for all the jobs which have an `"active_task_info"` object.